operations/agent-flow-mixin/dashboards/prometheus.libsonnet

```diff
@@ -105,13 +105,13 @@ local remoteWritePanels(y_offset) = [
     panel.withQueries([
       panel.newQuery(
         expr=|||
-          sum by (instance, component_id) (
-            prometheus_remote_storage_highest_timestamp_in_seconds{cluster="$cluster", namespace="$namespace", instance=~"$instance", component_id=~"$component"}
+          sum by (instance, component_path, component_id) (
+            prometheus_remote_storage_highest_timestamp_in_seconds{cluster="$cluster", namespace="$namespace", instance=~"$instance", component_path=~"$component_path", component_id=~"$component"}
             - ignoring(url, remote_name) group_right(instance)
-            prometheus_remote_storage_queue_highest_sent_timestamp_seconds{cluster="$cluster", namespace="$namespace", instance=~"$instance", component_id=~"$component", url=~"$url"}
+            prometheus_remote_storage_queue_highest_sent_timestamp_seconds{cluster="$cluster", namespace="$namespace", instance=~"$instance", component_path=~"$component_path", component_id=~"$component", url=~"$url"}
           )
         |||,
-        legendFormat='{{instance}} / {{component_id}}',
+        legendFormat='{{instance}} / {{component_path}} {{component_id}}',
```

Comment:
nit: the full path is `{{component_path}}/{{component_id}}`, should we add the `/` character to combine them rather than using a space to make it easier to read? (Ditto with the other legends)

Reply:
I tried that initially, but it looked weird for components on the root controller (`//prometheus.remote_write.default`). 
Maybe should write some jsonnet to make it pretty? 😅

Reply:
> I tried that initially, but it looked weird for components on the root controller (//prometheus.remote_write.default).
Maybe should write some jsonnet to make it pretty? 😅


🤔 Why is that what components in the root controller look like? It should be `/prometheus.remote_write.default` for rooted components, where `component_path=/`  and `component_id=prometheus.remote_write.default`

Reply:
Oh I understand now

Reply:
Sorry about that. I think it's fine for us to leave this as is and we can figure it out later. I don't think we can write Jsonnet to handle this since all the rendering is being done in Grafana at visualization time.

Marking as resolved :)